tests/sources/eltwise_unary_datacopy_test.cpp

```diff
@@ -19,18 +19,38 @@ uint32_t math_sync_tile_dst_index = 0;
 
 #include "llk_unpack_A.h"
 #include "llk_unpack_common.h"
+#include "llk_unpack_tilize.h"
 #include "params.h"
 
 void run_kernel()
 {
-    _llk_unpack_A_init_<BroadcastType::NONE, false, EltwiseBinaryReuseDestType::NONE, unpack_to_dest>(
-        0, 0, FACE_R_DIM, num_faces, formats.unpack_src, formats.unpack_dst);
-    _llk_unpack_A_hw_configure_<is_fp32_dest_acc_en, StochRndType::None>(formats.unpack_src, formats.unpack_dst, FACE_R_DIM, 0, num_faces);
-
-    for (int i = 0; i < TILE_CNT; ++i)
+    if (!tilize_en)
```

Comment:
this could be
```suggestion
    if constexpr (!tilize_en)
```
We discussed this offline. As you mentioned, using another set of API in case tilize is true might be hiding an issue. We need to investigate it.

Reply:
just fyi @amokanTT, when unpackA + tilize = true, it mean llk_pack also has to be tilize=true. That feature was added to be used together, and it was the way to implement faster tilize of a single tile on BH. tilize flag has to be consistent for both APIs